LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) {year} {author}
+Copyright (c) Trey Shugart <treshugart@gmail.com> 2016
```

Comment:
Interesting. That's a bit odd. I didn't touch the LICENSE template or file manually.